### PR TITLE
Authorization parameters for WalletRegistry contract

### PR DESCRIPTION
**Pull request by pdyraga**
Refs: #2834 

Introduced two authorization-related parameters in the WalletRegistry contract:
- minimum authorization
- authorization decrease delay

`Authorization` library will be further developed to enable integration with Threshold `TokenStaking` contract. Changes introducing governable parameters are included separately in this PR to keep the diff size sane.

Side note: after looking at `WalletRegistryGovernance`, I've decided to open https://github.com/keep-network/keep-core/issues/2847 to improve how governance contract works for Random Beacon and ECDSA.

### CODE REVIEW

File: solidity/ecdsa/contracts/WalletRegistry.sol [hunk 1]
```diff
@@ -170,6 +178,28 @@ contract WalletRegistry is IRandomBeaconConsumer, Ownable {
...
     }
 
+    /// @notice Updates the values of authorization parameters.
+    /// @dev Can be called only by the contract owner, which should be the
+    ///      wallet registry governance contract. The caller is responsible for
+    ///      validating parameters.
+    /// @param _minimumAuthorization New minimum authorization amount
+    /// @param _authorizationDecreaseDelay New authorization decrease delay in
+    ///        seconds
+    function updateAuthorizationParameters(
```
Comment (nkuba) [inline solidity/ecdsa/contracts/WalletRegistry.sol:188]: We should initialize the values for deployment in the constructor.

Comment (pdyraga) [reply solidity/ecdsa/contracts/WalletRegistry.sol:190]: Good point, see the two recent commits please.

File: solidity/ecdsa/contracts/WalletRegistryGovernance.sol [hunk 2]
```diff
@@ -447,6 +542,33 @@ contract WalletRegistryGovernance is Ownable {
...
+        view
+        returns (uint256)
+    {
+        return
+            getRemainingChangeTime(
+                minimumAuthorizationChangeInitiated,
+                CRITICAL_PARAMETER_GOVERNANCE_DELAY
+            );
+    }
+
```
Comment (dimpar) [inline solidity/ecdsa/contracts/WalletRegistryGovernance.sol:559]: Can we add a comment for this function as well?

Comment (pdyraga) [reply solidity/ecdsa/contracts/WalletRegistryGovernance.sol:559]: Yes, good catch! Please see https://github.com/keep-network/keep-core/pull/2848/commits/c68ce675c83f6061a92380a3f3723dbaa1e1c91b

**Review by dimpar — APPROVED** [review]

**Review by nkuba — APPROVED** [review]

**Pull request merged**